st2client/st2client/commands/pack.py

```diff
@@ -190,12 +190,41 @@ def __init__(self, resource, *args, **kwargs):
                                  help='Force pack installation.')
 
     def run(self, args, **kwargs):
+        if args.packs:
+            if len(args.packs) == 1:
+                args.pack = args.packs[0]
+                pack_info = self.manager.search(args, **kwargs)
+                content = getattr(pack_info, 'content', {})
+                if content:
+                    print '\nFor "%s" pack following content will be registered:\n' % args.pack
+                    for entity in content.keys():
+                        if entity != 'tests':
+                            print '%s:  %s' % (entity, content[entity]['count'])
+                    print '\nIt may take a while based on number of items in content.'
+            else:
+                pack_content = {}
+                for pack in args.packs:
+                    args.pack = pack
+                    pack_info = self.manager.search(args, **kwargs)
+                    content = getattr(pack_info, 'content', {})
+                    for entity in content.keys():
+                        if entity != 'tests':
+                            if entity not in pack_content:
+                                pack_content[entity] = content[entity]['count']
+                            else:
+                                pack_content[entity] += content[entity]['count']
+                if pack_content:
+                    print '\nFor "%s" packs following content will be registered:\n' \
+                        % ', '.join(args.packs)
+                    for item, count in pack_content.items():
+                        print '%s:  %s' % (item, count)
+                    print '\nIt may take a while based on number of items in content.'
```

Comment:
Minor compatibility thing - please use `print` as a function (use params) so it also works under Python 3.